http/vibe/http/client.d

```diff
@@ -884,6 +885,18 @@ final class HTTPClientResponse : HTTPResponse {
 		return m_maxRequests;
 	}
 
+
+	/// All cookies that shall be set on the client for this request
+	override @property Cookie[string] cookies() {
+		if ("Set-Cookie" in this.headers && cookies is null) {
```

Comment:
this is recursive, it should check m_`cookies` instead

Reply:
Oops, I'll fix that.